src/Image.re

```diff
@@ -77,3 +77,31 @@ external make:
   ) =>
   React.element =
   "Image";
+
+type error;
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external getSize:
+  (
+    ~uri: string,
+    ~success: (~width: float, ~height: float) => unit,
+    ~failure: error => unit=?,
+    unit
+  ) =>
+  unit =
+  "getSize";
+
+type requestId;
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external prefetch: (~uri: string) => requestId = "prefetch";
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external abortPrefetch: requestId => unit = "abortPrefetch";
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external queryCache: (~urls: array(string)) => unit = "queryCache";
```

Comment:
We should be consistent in whether we use `uri` or `url`. (Please also check what is used in other components/apis.)